components/definitions/chromel.d.ts

```diff
@@ -131,7 +131,7 @@ declare namespace chrome.braveRewards {
     addListener: (callback: (result: number, type: number) => void) => void
   }
 
-  const getExternalWallet: (callback: (result: number, wallet: RewardsExtension.ExternalWallet) => void) => {}
+  const getExternalWallet: (callback: (result: number, wallet?: RewardsExtension.ExternalWallet) => void) => {}
```

Comment:
Not 100% sure, but from what I can see, we omit `result` everywhere. Do we still need it?

Reply:
I believe so - I think we're using it on the reducer for the rewards page.